backend/service/project/project.go

```diff
@@ -0,0 +1 @@
+package project
```

Comment:
accidental? i guess it's the same as any other boilerplate

Reply:
Ah yeah I was working on the implementation ill back this out for now